index.html

```diff
@@ -62,23 +62,28 @@ <h3>Favourite games</h3>
        
         <h3>Haiku</h3>
 
-            A world of dew,<br><br>
-            And within every dewdrop,<br><br>
-            A world of struggle.<br><br>
-        <a><cite>by <span class="author">Kobayashi Issa</span></cite></a>
+        <p>
+            A world of dew,<br>
+            And within every dewdrop,<br>
+            A world of struggle.<br>
+            <cite>by <span class="author">Kobayashi Issa</span></cite>
+        </p>
 
     </main>
 
     <footer>
 
-        <hr><blockquote>The game is fun, if it's not fun, why bother?</blockquote>
-        <cite><a><em><span class="author">—Reggie Fils-Aimé</span></em></a></cite>
-        <p>© 2019 <a href="mailto:Jleveille05@gmail.com?subject=Message%20from%20CSD120%20Website">Jason Leveille</a></p>
+        <hr>
+        <blockquote>
+            The game is fun, if it's not fun, why bother?<br>
+            <cite>—<span class="author">Reggie Fils-Aimé</span></cite>
+        </blockquote>
+        <p>© 2020 <a href="mailto:Jleveille05@gmail.com?subject=Message%20from%20CSD120%20Website">Jason Leveille</a></p>
             <!--Found me!-->
 
     </footer>
 
-        <a href="◤">Back to the top</a>
+        <a href="#top" title="Back to top">◤</a>
```

Comment:
Use document fragment URL
*Text* of the link should be the triangle shape
*Title* of the link should be "Back to top"